sql/src/main/java/io/crate/analyze/PartitionPropertiesAnalyzer.java

```diff
@@ -84,6 +95,34 @@ public static PartitionName toPartitionName(DocTableInfo tableInfo,
         return new PartitionName(tableInfo.ident(), Arrays.asList(values));
     }
 
+    public static PartitionName toPartitionName(DocTableInfo tableInfo,
```

Comment:
as i mentioned here https://github.com/crate/crate/pull/9194#discussion_r329813746, but we can probably do it later. i'll just then wait until this PR is merged to adjust #9194